### ISSUE TITLE
Road Tools Upgrade

### ISSUE THREAD

**Issue by anthonyromyn** [DuncanLab/RoadRAGE#26]
Hi Cody,

Hope all is well. Looking around for some improved road tools, I settled on trying to use MicroGSD's Road Architecture (https://github.com/MicroGSD/RoadArchitect). I can get it working nicely on it's own with some random terrain, but I'm having trouble implementing it in the roadrage prefab maps (e.g. road_grass_400m_WP). Might just be me being inexperienced with Unity, and I'll keep trying at it, but wanted to give you an update. 

Best,
Anthony

**Comment by anthonyromyn** [comment]
Hi Cody,

I've made some progress but still running into one issue in particular. When using RoadArchitect, certain pieces aren't loading or are only partially loaded. I've recorded a video of it here (https://vimeo.com/325892664 p/w "RoadRage") if you'd like to see it. A few things that stand out to me that you'll see in the video:
1) When I load the prefab, the road and concrete divider don't load until I hit the update road button. Then, if I save and reload, the same behaviour occurs.
2) Then, when running RoadRAGE, we see that the same road and concrete divider objects aren't loading. I assume these are related, but haven't yet been able to solve. 

Any ideas?
Thanks!
Anthony
